google_auth_oauthlib/helpers.py

```diff
@@ -125,14 +126,23 @@ def credentials_from_session(session, client_config=None):
             "There is no access token for this session, did you call " "fetch_token?"
         )
 
-    credentials = google.oauth2.credentials.Credentials(
-        session.token["access_token"],
-        refresh_token=session.token.get("refresh_token"),
-        id_token=session.token.get("id_token"),
-        token_uri=client_config.get("token_uri"),
-        client_id=client_config.get("client_id"),
-        client_secret=client_config.get("client_secret"),
-        scopes=session.scope,
-    )
+    if client_config.get("3pi"):
```

Comment:
how is this 3pi config created/obtained?

Reply:
When we create a Flow, we supply it with a client_config.  If we intend for the Flow to use 3PI, it will now look like this:

```
{
  "web": {
    "client_id": "asdfjasdljfasdkjf",
    "client_secret": "1912308409123890",
    "redirect_uris": ["https://www.example.com/oauth2callback"],
    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
    "token_uri": "https://accounts.google.com/o/oauth2/token",
    "3pi": true
  }
}
```